tiledb/sm/array_schema/domain.cc

```diff
@@ -918,6 +918,8 @@ void Domain::set_tile_cell_order_cmp_funcs() {
         cell_order_cmp_func_2_[d] = nullptr;
         break;
       case Datatype::BLOB:
+      case Datatype::GEOM_WKB:
+      case Datatype::GEOM_WKT:
       case Datatype::CHAR:
       case Datatype::BOOL:
       case Datatype::STRING_UTF8:
```

Comment:
Optional: For the invalid dimension types would it be safer to throw and error than set all the pointers to nullptr?